lab/skel_graph/traversal/problems/alee/alee.cpp

```diff
@@ -0,0 +1,152 @@
+// Copyright 2020 Nicolae Mara-Ioana <maraioana9967@gmail.com>
```

Comment:
Please remove this file and apply the suggestions made in main.cpp, as @neatudarius you have duplicate file